src/core/QueryManager.ts

```diff
@@ -572,7 +572,7 @@ export class QueryManager<TStore> {
           definitions: transformed.definitions.map(def => {
             if (def.kind === "OperationDefinition" &&
                 def.operation !== "query") {
-              return { ...def, operation: "query" };
+              return { ...def, operation: "query" as OperationTypeNode };
```

Comment:
We can use the enum value directly here

```suggestion
              return { ...def, operation: OperationTypeNode.QUERY };
```

Reply:
Given that `OperationTypeNode` doesn't exist in GraphQL 15, this is fine how it is